src/configure/helper/repoAnalysisHelper.ts

```diff
@@ -46,32 +47,35 @@ export class RepoAnalysisHelper {
                 let applicationSettings: RepositoryAnalysisApplicationSettings = new RepositoryAnalysisApplicationSettings();
                 applicationSettings.language = analysis.language;
 
-                if(!!analysis.settings && !!analysis.buildTargetName) {
```

Comment:
rename `analysis` to `applicationSetting`

Reply:
we are constructing applicationSettings already, this will induce confusion.